cmd/agent/dist/conf.d/systemd.d/conf.yaml.example

```diff
@@ -14,7 +14,30 @@ instances:
     ## using Docker Agent (https://docs.datadoghq.com/agent/docker/).
     ##
     # private_socket: <PATH_TO_SYSTEMD_PRIVATE_SOCKET>
-    
+
+    ## @param substate_status_mapping - object - optional
+    ## Custom mapping of systemd substate for the 'systemd.unit.state' service check status.
```

Comment:
I think it's better to not replace the `systemd.unit.state` but create a new service check called `systemd.unit.substate` that uses the mapping. I think some users might want both.

Note: you can probably reuse `getServiceCheckStatus` twice.

Reply:
I don't really see a use case for having both but it doesn't hurt and is probably easier to work with. Also the change is minimal so going forward with that idea.